xcell/mappers/mapper_base.py

```diff
@@ -37,6 +38,14 @@ def get_contaminants(self):
         return None
 
     def get_mask(self):
+        if self.mask is None:
+            fn = '_'.join([f'mask_{self.mask_name}',
```

Comment:
Hmm, do you need a name for the mask? Why not the same name as for the other maps? (i.e. `self.map_name`)?

Reply:
In fact, if you like that, I think we can give a `self.map_name` to each mapper that it's fixed (or a default that can be changed)

Reply:
In other words, all tracers need a `mask_name` (otherwise initialization fails)